app/helpers.php

```diff
@@ -985,3 +985,8 @@ function group_users_by_online_state($users)
         'offline' => $offline,
     ];
 }
+
+function product_url($product)
+{
+    return route('store.product.show', ['product' => $product]);
```

Comment:
`route('store.products.show', $products)` _should_ work so this helper probably isn't needed.